cards.c

```diff
@@ -0,0 +1,85 @@
+/*Jinoo Hong*/
+/*Write a program `cards.c` that implements a deck of cards.  Implement the suits as an enum, and the cards as a struct containing the value and suit.  Implement a function `drawCard()` to generate a random card and `getSuit()` to print out the the suit properly.  In main, draw and print out 5 cards in the following format:*/
```

Comment:
Perfect! Just make sure to include comments before your functions saying what they do/return!